### PR TITLE
[SW-642] Write tests for Mojo predictions in python, fix bugs

### PR DESCRIPTION
**Pull request by jakubhava**
RIght now, H2OGBM behaves correctly in the consistent way in Scala and Python.

H2ODeeplearning still doesn't - created jira for it https://0xdata.atlassian.net/browse/SW-643

### CODE REVIEW

File: py/pysparkling/ml/algo.py [hunk 1]
```diff
@@ -9,11 +9,11 @@
 class H2OGBM(JavaEstimator, H2OGBMParams, JavaMLReadable, JavaMLWritable):
     @keyword_only
     def __init__(self, ratio=1.0, predictionCol=None, featuresCols=[], allStringColumnsToCategorical=True, nfolds=0,
-                 keepCrossValidationPredictions=False, keepCrossValidationFoldAssignment=False, parallelizeCrossValidation=False,
+                 keepCrossValidationPredictions=False, keepCrossValidationFoldAssignment=False, parallelizeCrossValidation=True,
```
Comment (jakubhava) [inline py/pysparkling/ml/algo.py]: This was bug, default value is True

File: py/pysparkling/ml/algo.py [hunk 2]
```diff
@@ -9,11 +9,11 @@
...
                  seed=-1, distribution="AUTO", ntrees=50, maxDepth=5, minRows=10.0, nbins=20, nbinsCats=1024, minSplitImprovement=1e-5,
                  histogramType="AUTO", r2Stopping=SparkSession.builder.getOrCreate()._jvm.Double.MAX_VALUE,
                  nbinsTopLevel=1<<10, buildTreeOneNode=False, scoreTreeInterval=0,
-                 sampleRate=0.632, sampleRatePerClass=None, colSampleRateChangePerLevel=1.0, colSampleRatePerTree=1.0,
+                 sampleRate=1.0, sampleRatePerClass=None, colSampleRateChangePerLevel=1.0, colSampleRatePerTree=1.0,
```
Comment (jakubhava) [inline py/pysparkling/ml/algo.py]: Default value for GBM is 1.0 for sampleRate

File: py/tests/tests_unit_conversions.py [hunk 3]
```diff
@@ -189,5 +190,21 @@ def test_sparse_data_conversion(self):
 
         assert total < 10 # The conversion should not take longer then 10 seconds
 
+    def test_load_mojo_gbm(self):
+        from pysparkling.ml import H2OMOJOModel, H2OGBM
+        mojo = H2OMOJOModel.create_from_mojo("../ml/src/test/resources/binom_model_prostate.mojo")
```
Comment (mmalohlava) [inline py/tests/tests_unit_conversions.py]: is this file in repo?

Comment (jakubhava) [reply py/tests/tests_unit_conversions.py]: yup, it's the same one we are using for tests in `H2OMojoTests`

**Pull request merged**